docs/contracts/deposit-security-module.md

```diff
@@ -255,65 +286,84 @@ Pauses deposits given that both conditions are satisfied (reverts otherwise):
  The signature, if present, must be produced for keccak256 hash of the following
  message (each component taking 32 bytes):
 
- | PAUSE_MESSAGE_PREFIX | blockNumber
+ | PAUSE_MESSAGE_PREFIX | blockNumber | stakingModuleId |
+
+If the staking module is not active does nothing.
+In case of an emergency, the function `pauseDeposits` is supposed to be called
+by all guardians. Thus only the first call will do the actual change. So
+the other calls would be OK operations from the point of view of the protocol logic.
 
 ```sol
-function pauseDeposits(uint256 blockNumber, Signature memory sig)
+function pauseDeposits(uint256 blockNumber, uint256 stakingModuleId, Signature memory sig)
 ```
 
-#### Parameters:
+#### Parameters
 
-| Name          | Type        | Description                                                                          |
-| ------------- | ----------- | ------------------------------------------------------------------------------------ |
-| `blockNumber` | `uint256`   | Block number with malicious pre-deposits have been observed by the guardian          |
-| `sig`         | `Signature` | Short ECDSA guardian signature as defined in https://eips.ethereum.org/EIPS/eip-2098 |
+| Name              | Type        | Description                                                                          |
+| ----------------- | ----------- | ------------------------------------------------------------------------------------ |
+| `blockNumber`     | `uint256`   | Block number with malicious pre-deposits have been observed by the guardian          |
+| `stakingModuleId` | `uint256`   | Id of the staking module to pause deposits for                                       |
+| `sig`             | `Signature` | Short ECDSA guardian signature as defined in https://eips.ethereum.org/EIPS/eip-2098 |
 
 ### unpauseDeposits()
-Unpauses deposits. Only callable by the owner.
-
-```sol
-function unpauseDeposits()
-```
 
-### setLastDepositBlock()
-Sets `lastDepositBlock`. Only callable by the owner.
+Unpauses deposits for staking module.
+If the staking module is not paused, do nothing.
 
 ```sol
-function setLastDepositBlock(uint256 newLastDepositBlock) external;
+function unpauseDeposits(uint256 stakingModuleId)
 ```
 
-#### Parameters:
+:::note
+Reverts if any of the following is true:
+- `msg.sender` is not the owner.
+:::
 
-| Name                  | Type        | Description                              |
-| --------------------- | ----------- | ---------------------------------------- |
-| `newLastDepositBlock` | `uint256`   | Block number containing the last deposit |
+#### Parameters
 
+| Name              | Type      | Description              |
+| ----------------- | --------- | ------------------------ |
+| `stakingModuleId` | `uint256` | Id of the staking module |
 
 ### depositBufferedEther()
-Calls Lido.depositBufferedEther(maxDepositsPerBlock).
+
+Calls LIDO.deposit(maxDepositsPerBlock, stakingModuleId, depositCalldata).
```

Comment:
Should probably, clarify here. Because, rn it looks like we just calling other contract function here and it's strange.

Reply:
fixed